src/components/Button/utils.ts

```diff
@@ -1,5 +1,7 @@
 import {ButtonSize, ButtonView} from '@gravity-ui/uikit';
 
+export const ICON_QA = 'button-icon';
```

Comment:
Why is ICON_QA here? Maybe it should be replaced to Button.test.tsx?

Reply:
It is used in both test and component files. Let him be here )